jobs/e2e_node/perf-image-config.yaml

```diff
@@ -0,0 +1,9 @@
+---
+images:
+  cosbeta-resource1:
+    image: cos-81-12871-103-0
+    project: cos-cloud
+    machine: n1-standard-16
+    metadata: "user-data<test/e2e_node/jenkins/gci-init.yaml,gci-update-strategy=update_disabled"
+    tests:
+      - 'Node Performance Testing'
```

Comment:
(this is me not fully understanding how all of this works)

Does this take precedence over the `[Flaky]` regex in the job? Work in combination with it?

It feels like we're overloading/misdirecting by using the term "flaky" when really this job seems to be running perf tests

Reply:
it works in combination with the focus on `[Flaky]`.

And it is overloading and misdirecting.
The long version of the story is in 
https://github.com/kubernetes/kubernetes/issues/91263
but the short version is that these tests were meant to run in an n1-standard-16 sized machine, see https://github.com/kubernetes/kubernetes/pull/65250.

But for some reason the e2e tests running in this current version of the job were never properly modified to run in that machine and have been running since the beginning in an n1-standard-1.
As such, the have always been failing.
Then they were labeled as "flaky" in https://github.com/kubernetes/kubernetes/pull/78548 with the intent of removing them from the kubelet-serial job.
The idea was to move them to their own e2e job so they didn't pollute another job.

My idea here was to explicitly specify within the image config that this whole fix is only for a handful of performance tests that were labeled as flaky becuase they were running in the wrong machine.
I think a better place for the tests in this job would be in https://github.com/kubernetes/test-infra/blob/master/jobs/e2e_node/benchmark-config.yaml but i guess one step at a time (?)

Reply:
It happens to be the only test marked flaky left in node_e2e.

I think the idea of both `Flaky` regex and `'Node Performance Testing'` is for this to be moved out of flaky and into it's own job eventually. (It should probably be it's own job already, because nothing else is backed by a `n1-standard-16`.

Reply:
ok, I won't block on this one as long as there's followup to either rename this job to something like `ci-kubernetes-node-kubelet-perf`, or split out a new job that does so... I think the node perf tests shouldn't be tagged as `[Flaky]` but instead something like `[Feature:Node Performance]`

we currently have no way of verifying/linking whether all e2e tests are executed by at least one CI job, so I could understand if someone wants to preemptively keep a flaky job around even if there are no flaky tests

Reply:
> ok, I won't block on this one as long as there's followup to either rename this job to something like ci-kubernetes-node-kubelet-perf, or split out a new job that does so... I think the node perf tests shouldn't be tagged as [Flaky] but instead something like [Feature:Node Performance]

YES to all of this! probably split out a new job for this (or move the contents of the image config into the jobs running cenchmarks TBD with SIG Node).
And huge +1 on `[Feature:Node Performance]`